### PR TITLE
Ts migration: ExistingWorkspaceForm

### PR DESCRIPTION
**Pull request by imolina212**
Updated Types to MUI specific types "Sx Props"

Notes:

- Changed SupportIconsBar and ButtonAppBar to .tsx
- investigating tab space issue in PR (it doesn't have extra spacing in local branch)
- Grouped MUI imports for cleaner code

Developer's Certificate of Origin 1.1

        By making a contribution to this project, I certify that:

        (a) The contribution was created in whole or in part by me and I
            have the right to submit it under the open source license
            indicated in the file; or

        (b) The contribution is based upon previous work that, to the best
            of my knowledge, is covered under an appropriate open source
            license and I have the right under that license to submit that
            work with modifications, whether created in whole or in part
            by me, under the same open source license (unless I am
            permitted to submit under a different license), as indicated
            in the file; or

        (c) The contribution was provided directly to me by some other
            person who certified (a), (b) or (c) and I have not modified
            it.

        (d) I understand and agree that this project and the contribution
            are public and that a record of the contribution (including all
            personal information I submit with it, including my sign-off) is
            maintained indefinitely and may be redistributed consistent with
            this project or the open source license(s) involved.

### CODE REVIEW

**Comment by imolina212** [comment]
Migrated ExistingWorkspaceForm to tsx

Notes:

- changed handleClick prop to ()=> void because of ButtonIBM

**Comment by arielge** [comment]
@imolina212 Welcome to the Label Sleuth community, and thank you for contributing!

Now, I ran our frontend test workflow and it appears these changes result in some errors. Note that you can try building the frontend on your machine by following the instructions [here](https://github.com/label-sleuth/label-sleuth/tree/main/frontend), and that you can run the tests with `npm run test`. @martinscooper should be able to help you out if you have any questions.

File: frontend/src/components/bars/upperBar/ButtonAppBar.tsx
Comment (martinscooper) [inline frontend/src/components/bars/upperBar/ButtonAppBar.tsx:1]: This file has been already migrated too.

File: frontend/src/modules/Workspace-config/ExistingWorkspaceForm.tsx
Comment (martinscooper) [inline frontend/src/modules/Workspace-config/ExistingWorkspaceForm.tsx:1]: Good work, this files has been correctly migrated and can be merged!

**Comment by imolina212** [comment]
@arielge Thanks for letting me know!

**Pull request closed without merging**